relay-metrics/src/statsd.rs

```diff
@@ -183,3 +193,13 @@ pub fn metric_name_tag(value: &str) -> &str {
 
     "other"
 }
+
+/// Returns the metric type for use as a tag key on statsd metrics.
+pub(crate) fn metric_type_tag(value: &BucketValue) -> &'static str {
```

Comment:
iiuc, this is useful for the cases where the metric name tag resolves to "other"? as otherwise the first letter would give the type?

Reply:
Yep!